Author: Chris Herborth <chris.herborth@chainguard.dev>
Forwarded: not-needed
Description: Fixes the following CVEs in json and net-imap:
  - CVE-2026-33210 GHSA-3m6g-2423-7cp3
  - CVE-2026-42245 GHSA-q2mw-fvj9-vvcw
  - CVE-2026-42246 GHSA-vcgp-9326-pqcp
  - CVE-2026-42256 GHSA-87pf-fpwv-p7m7
  - CVE-2026-42257 GHSA-hm49-wcqc-g2xg
  - CVE-2026-42258 GHSA-75xq-5h9v-w6px
  - CVE-2026-47240 GHSA-8p34-64r3-mwg8
  - CVE-2026-47241 GHSA-c4fp-cxrr-mj66
  - CVE-2026-47242 GHSA-46q3-7gv7-qmgg
  - CVE-2026-54696 GHSA-x2f5-4prf-w687
diff --git a/lib/pom.rb b/lib/pom.rb
index 7d66daf1f6..e7ea75b749 100644
--- a/lib/pom.rb
+++ b/lib/pom.rb
@@ -51,7 +51,7 @@ default_gems = [
   ['jar-dependencies', '0.5.7'],
   ['jruby-readline', '1.3.7'],
   ['jruby-openssl', '0.16.2'],
-  ['json', '2.18.0'],
+  ['json', '2.19.9'],
   ['net-http', '0.9.1'],
   ['net-protocol', '0.2.2'],
   ['open-uri', '0.5.0'],
@@ -114,7 +114,7 @@ bundled_gems = [
   ['minitest', '6.0.0'],
   ['mutex_m', '0.3.0'],
   ['net-ftp', '0.3.9'],
-  ['net-imap', '0.6.2'],
+  ['net-imap', '0.6.4.1'],
   ['net-pop', '0.1.2'],
   ['net-smtp', '0.5.1'],
   ['nkf', '0.2.0'],
